test/functionalTest/cases/2015_notification_templates/notification_templates.test

```diff
@@ -337,7 +337,7 @@ User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
 Ngsiv2-Attrsformat: normalized
 Host: 127.0.0.1:REGEX(\d+)
 Accept: application/json
-Content-Type: application/json; charset=utf-8
+Content-Type: text/plain
```

Comment:
> Note this notification is Ngsiv2-Attrsformat: normalized. Thus Content-Type: application/json; charset=utf-8 should be used.

Looking to the code (https://github.com/telefonicaid/fiware-orion/pull/2172/files#diff-4203db90bc90ad52ab2e6fbeccfdac1eR109) the logic seems to be correct. This is werid... Maybe some "sorting effect" due to SORT_START / SORT_END ?